src/main/java/com/basho/riak/client/core/RiakNode.java

```diff
@@ -640,18 +650,23 @@ private Channel getConnection()
         {
             try
             {
-                channel = doGetConnection();
+                channel = doGetConnection(true);
```

Comment:
Not that I can see, the only places we call doGetConnection/1 are in [start()](https://github.com/basho/riak-java-client/blob/fixes/am/572/src/main/java/com/basho/riak/client/core/RiakNode.java#L268), [the general getConnection/1 method](https://github.com/basho/riak-java-client/blob/fixes/am/572/src/main/java/com/basho/riak/client/core/RiakNode.java#L653), and [the health checker](https://github.com/basho/riak-java-client/blob/fixes/am/572/src/main/java/com/basho/riak/client/core/RiakNode.java#L1127).